docs/source/classroom/class_guide.md

```diff
@@ -0,0 +1,172 @@
+(class-guide)=
+# Courses on HPC at NU
+
+We support classroom education at Northeastern University by providing access to computing resources (CPU and GPU) and storage resources for instructors and their students.
+
+We’ve supported courses from many disciplines, including biology, chemistry, civil engineering, machine learning, computer science, mathematics, and physics.
+
+To gain access to HPC resources instructors need to submit a [classroom access form.](https://service.northeastern.edu/tech?id=sc_cat_item&sys_id=0ae24596db535fc075892f17d496199c)
+
+Please submit these requests prior to the beginning of each semester (preferred), or at least **one week** prior to the start of when you plan on using the HPC cluster for your class.
+
+## Classroom setup
+---
+
+Once access is provided, each course will have a course-specific directory under `/courses/` following this sample file tree. As shown for the course BINF6430.202410 below:
+
+:::{code-block} bash
+
+/courses/
+└── BINF6430.202410/
+    ├── data/
+    ├── staff/
+    └── students/
+
+:::
+
+The sub-directory `staff/` will be populated with a folder for each of the following: instructors, co-instructors, and TAs. The `students/` sub-directory contains a folder for each student. And the `data/` sub-directory can be populated by those in staff but is read-only for students.
+
+All those in staff have read-write-execute permissions within the entirety of their courses directory, allowing them to store data, homework assignments, build conda environments, create new directories, etc, as they see fit.
```

Comment:
We might want to add a bit saying the student directories are only enterable by the student's themselves and they cannot look into other student directories out of the box. It might be implicit but being explicit might be worthwhile.